supertest/functions/biggestboi/config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "biggestboi",
```

Comment:
Is this supposed to be here at all ? 

I call Sus!

Reply:
😅